src/redux/reducers/customLabels.js

```diff
@@ -0,0 +1,48 @@
+import {
+  GET_CUSTOM_LABELS_SUCCESS,
+  GET_CUSTOM_LABELS_FAILURE,
+  GET_CUSTOM_LABELS,
+} from '../actions';
+
+import { formatErrors } from '../helpers';
+
+const handlers = {
+  [GET_CUSTOM_LABELS]: (state) => {
+    return {
+      ...state,
+      isLoading: true,
+    };
+  },
+  [GET_CUSTOM_LABELS_SUCCESS]: (state, action) => {
+    return {
+      ...state,
+      isLoading: false,
+      items: action.payload.customLabels,
+    };
+  },
+  [GET_CUSTOM_LABELS_FAILURE]: (state, action) => {
+    const {
+      payload: { errors },
+    } = action;
+
+    return {
+      ...state,
+      isLoading: false,
+      errors: formatErrors(errors),
```

Comment:
Please check the format of errors, returned from the cutomLables endpoint to be sure that it is properly hanled by  formatErrors func